src/specshell.software.omnic.dde/Commands/Multiply.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class Multiply : IDdeExecuteCommand
     {
```

Comment:
does this default to 0f anyways :P ?

Reply:
Checks should be applied else where :)
By not setting a default value it requires the developer to pass a value :)